### PR TITLE
Fixed Mobile UI wrapping

### PR DESCRIPTION
**Pull request by anthonyoliai**
I'm still kind of new to React and web development itself, but I wanted to help the honey community, so I started working on some simple issues. This should properly wrap the proposal text once the window's width gets too small. 

EDIT: I also noticed that the progress bar, is not wrapping properly. In Desktop mode, when the resolution gets small enough, the progress bar doesn't wrap properly. This PR also fixes that issue. Just a quick observation!

### CODE REVIEW

**Comment by sembrestels** [comment]
Thanks @anthonyoliai for taking the time for doing this!

File: src/screens/Proposals.js [hunk 1]
```diff
@@ -265,6 +265,9 @@ const IdAndTitle = ({ id, name, selectProposal }) => {
       <span
         css={`
           color: ${theme.surfaceContentSecondary};
+          white-space:normal;
+          float:left;
+          text-align:left;
```
Comment (fabriziovigevani) [inline src/screens/Proposals.js:270]: What do you think if we apply this styles to the parent `Link` component? In that case we can drop the `float` prop.

**Review by fabriziovigevani — COMMENTED** [review]
Thanks @anthonyoliai for this! Left one suggestion.

Also you mentioned you did a fix on the progress bar that you might have forgotten to push 😄

File: src/screens/Proposals.js (hunk 1 above)
Comment (anthonyoliai) [reply src/screens/Proposals.js:270]: Hi , yes sorry i'll try to get it figured out. Sorry still new to using github 😅

Comment (fabriziovigevani) [reply src/screens/Proposals.js:270]: haha no to worry!  Thank you

Comment (anthonyoliai) [reply src/screens/Proposals.js:270]: Haha alright. I'm still learning as i'm going, as far as I can see you recommended me to apply the style to the parent Link component. I've however been having a hard time finding this as I am new to the React infrastructure. How I see it is that the modified Link component is part of `const IdAndTitle` , which is in turn part of a large DataView `Proposals.js line 185-235 ` I've been trying a couple of things out applying style changes for different components to no success.  I feel like I'm not looking at the right place, tips would be very appreciated as all of this is a learning process for me :)

Comment (fabriziovigevani) [reply src/screens/Proposals.js:270]: Ah sorry, maybe i should have been more specific! I was referring to do something like this:

```
    <Link
      onClick={handleOnClick}
      css={`
        white-space: normal;
        text-align: left;
      `}
    >
      <span
        css={`
          color: ${theme.surfaceContentSecondary};
        `}
      >
        {name}
      </span>
    </Link>
```

Comment (anthonyoliai) [reply src/screens/Proposals.js]: Done! Sorry for the wait. Also, by taking this styling approaches the progress bar is automatically fixed aswell, so for that nothing else had to be pushed. To recreate: On Desktop mode when resizing the screen resolution of the browser, the progress bar will wrap properly.

File: src/screens/Proposals.js [hunk 2]
```diff
@@ -261,7 +261,13 @@ const IdAndTitle = ({ id, name, selectProposal }) => {
   }, [id, selectProposal])
 
   return (
-    <Link onClick={handleOnClick}>
+    <Link
+      onClick={handleOnClick}
+      css={`
+      white-space: normal;
+      text-align: left;
+    `}
```
Comment (fabriziovigevani) [inline src/screens/Proposals.js:269]: Ah, could we fix the formatting? 

```suggestion
      css={`
        white-space: normal;
        text-align: left;
      `}
```

Comment (anthonyoliai) [reply src/screens/Proposals.js]: Done! Sorry been a long day.

Comment (rperez89) [reply src/screens/Proposals.js]: ah this formatting stuff might be fixed when you save the changes to the file thanks to the .eslint configuration, if that is not working to you let us know

**Review by rperez89 — COMMENTED** [review]
nice!!

**Review by fabriziovigevani — APPROVED** [review]
🙌  Thanks @anthonyoliai !

**Pull request merged**